core/designsystem/src/main/java/com/droidknights/app2023/core/designsystem/component/TopAppBar.kt

```diff
@@ -55,6 +58,11 @@ fun KnightsTopAppBar(
             modifier = Modifier
                 .fillMaxWidth()
                 .background(containerColor)
+                .clickable(
+                    interactionSource = remember { MutableInteractionSource() },
+                    indication = null,
+                    onClick = {}
+                )
```

Comment:
클릭 이벤트 투과 방지 목적이라면 `pointerInput`을 사용하여 포인터 입력 이벤트를 정의하면 어떨까요?
```suggestion
                .pointerInput(Unit) {}
```

KnightsTopAppBar는 클릭 가능하도록 설계할 필요가 없어서 `clickable`을 사용하는 것보다 더 명시적이지 않을까 합니다. `clickable` 내부에서 `pointerInput`을 사용하고 있기도 하고요.

참고로 Surface 컴포저블을 뜯어보면 동일한 로직이 포함되어 있습니다.
```kotlin
fun Surface( /* ... */ ) {
    Box(
        // ...
        .pointerInput(Unit) {},
    ) {
    // ...
}

Reply:
pointerInput을 사용할 생각을 못했네요 감사합니다 수정하도록 하겠습니다 😃